uint8 robot_id
string robot_state
